packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -60,6 +60,10 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
 
   const threads = sortPinned(sortByFeaturedFilter(data || [], featuredFilter));
 
+  useEffect(() => {
+    document.title = `${app.chain.meta.name} – Discussions`;
+  }, []);
```

Comment:
in the past the proper way of doing this was using [react-helmet](https://www.npmjs.com/package/react-helmet) but nowadays in the hooks-era this solution is fine. The better solution though would be to have a custom hook like eg`useManageDocumentTitle` and it should be used everywhere where we want to set custom title, instead of using useEffect multiple times in multiple components.

Reply:
Noted, I'll make a hook for that.

Reply:
A hook with the name you suggested was made for setting the page title.
It can receive either:
- the title as a string
- a fallback string to be set, a component state variable, its attribute where the title can be read (e.g. `thread.title`). The last two are optional parameters